src/components/MarkersDisplay/MarkersDisplay.md

```diff
@@ -18,6 +18,7 @@ This component reads manifest data from central state management provided by Con
 `MarkersDisplay` component allows the following props;
 - `showHeading`: accepts a Boolean value, which has a default value of `true` and is _not required_. This enables to hide the `Markers` heading on top of the component allowing to customize the user interface.
 - `headingText`: accepts a String value, which has a default value of `Markers` and is _not required_. This value is used in the heading of the component, and enables to customize the text.
+- `displayMotivations`: accepts an Array including a list of [supported motivations](https://iiif.io/api/presentation/3.0/#values-for-motivation) for Annotation type resources in IIIF Presentation 3.0 spec, which has a default value of `[]` in which case will display all annotations related to the Canvas and is _not required_. For playlist manifests, Ramp sets this value to `['highlighting']` overwriting any given set of values as this component is intented to markers in playlists.
```

Comment:
Is it worth calling out that only tagging, commenting, and supplementing motivations are handled at the moment?  (At least this is what it looks like to me in `AnnotationRow`.)